### PR TITLE
Python interop

### PR DESCRIPTION
**Pull request by dankeyy**
Adding a python-interop example
Feel free to comment any improvements :)

### CODE REVIEW

File: examples/python-interop/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Python Interop
+
+This is an example of calling Roc code from [Python](https://www.python.org/).
+
+## Installation
+
+The following was tested on NixOS, with Python 3.10, clang 13.0.1, gcc 11.3.0 but this should work on with most recent python3 and clang versions on most modern Linux and MacOS.\
+Of course you're welcome to test on your machine and tell me if you ran into any issues or limitations.
```
Comment (rtfeldman) [inline examples/python-interop/README.md:8]: Someone coming into the repo fresh won't know who "me" is - maybe include a link to DM you on Zulip? 😄

**Review by rtfeldman — APPROVED** [review]
One suggestion, but overall looks good - very exciting to have a working Python interop example, thanks @dankeyy! 😃

File: examples/python-interop/README.md (hunk 1 above)
Comment (dankeyy) [reply examples/python-interop/README.md]: Right, added a link, commited below

**Review by rtfeldman — APPROVED** [review]
Awesome, thanks for the revisions!

**Pull request merged**